csi/csi.go

```diff
@@ -43,12 +44,13 @@ type OsdCsiServerConfig struct {
 // proxies CSI requests for a single specific driver
 type OsdCsiServer struct {
 	Server
-	listener net.Listener
-	server   *grpc.Server
-	driver   volume.VolumeDriver
-	wg       sync.WaitGroup
-	running  bool
-	lock     sync.Mutex
+	listener    net.Listener
```

Comment:
Please add comments for member.

Reply:
Not sure what you mean, could you show me an example? or are you saying we should add comments from now on for all osd structs?

Reply:
https://github.com/libopenstorage/openstorage/blob/master/api/api.go

Reply:
I see some done and some not. This is an internal struct, does it need to be commented in that manner? I agree with you that `OsdCsiServerConfig` should definitely have each member commented since it is used by the caller.

Reply:
Created #224